### PR TITLE
its to it's where appropriate in MessageTextBox.Generated.cs

### PR DESCRIPTION
**Pull request by mattdbr**

### CODE REVIEW

File: src/Microsoft.PowerShell.GraphicalHost/ManagementList/Common/MessageTextBox.Generated.cs [hunk 1]
```diff
@@ -20,7 +20,7 @@ namespace Microsoft.Management.UI.Internal
 {
     
     /// <summary>
-    /// A TextBox which shows a user provided text when its empty.
+    /// A TextBox which shows a user provided text when it's empty.
```
Comment (daxian-dbw) [inline src/Microsoft.PowerShell.GraphicalHost/ManagementList/Common/MessageTextBox.Generated.cs]: This file looks like auto-generated, so we probably should skip this change.

**Comment by akkatracker** [comment]
Fair enough. Is this autogenerated by code in the Powershell Repo? If so is there any way to fix it?

**Comment by daxian-dbw** [comment]
>//  <auto-generated>
//      This code was generated by a tool. DO NOT EDIT
//
//      Changes to this file may cause incorrect behavior and will be lost if
//      the code is regenerated.
//  </auto-generated>
// -----------------------------------------------------------------------

@lzybkr @KrishnaV-MSFT @daviwil @SteveL-MSFT does anyone know how the *.Generated.cs files were generated? History in source depot shows those files haven't been changed for more than 7 years. If we are not going to re-generate them in future, maybe we should remove the confusing comments.

**Comment by daviwil** [comment]
I think this code might have been generated in the past but its contents look hand-edited judging from the commits I see.  Not sure why the "DO NOT EDIT" comment was left in.  Seems like it's fine to accept this change to fix the comment!

**Comment by daxian-dbw** [comment]
Thanks @daviwil , then this PR LGTM.

**Comment by daxian-dbw** [comment]
Issue #2141 was opened to track the obsolete comments.

**Pull request merged**